artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/Message.java

```diff
@@ -80,6 +80,8 @@ public interface Message {
    // The value is somewhat higher on 64 bit architectures, probably due to different alignment
    int memoryOffset = 352;
 
+   int ESTIMATE_DATA_WEIGHT = 2;
```

Comment:
I generally like to see values like this carry some amount of documentation about what they do and why they have the given value just so future maintainers don't have to spend time to come up with an educated guess.

Reply:
sure.. thanks!

I also moved it to AMQPMessage.